### PR TITLE
Reset the Optional for each test.

### PR DESCRIPTION
**Pull request by shanemc-goog**
### Description
TearDown may happen when unnecessary otherwise.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

**Review by clouser2 — APPROVED** [review]

File: server/test/modules/MainModuleTest.java [hunk 1]
```diff
@@ -29,6 +30,11 @@ public class MainModuleTest {
   // This means that there's no need or a corresponding @Before method.
   private Optional<Application> maybeApp;
 
+  @Before
+  public void setUpApp() {
+    maybeApp = Optional.empty();
+  }
+
```
Comment (shanemc-goog) [inline server/test/modules/MainModuleTest.java:37]:
```suggestion

```

File: server/test/modules/MainModuleTest.java [hunk 2]
```diff
@@ -29,6 +30,10 @@ public class MainModuleTest {
   // This means that there's no need or a corresponding @Before method.
   private Optional<Application> maybeApp;
 
+  @Before
+  public void setUpApp() {
+    maybeApp = Optional.empty();
+  }
```
Comment (shanemc-goog) [inline server/test/modules/MainModuleTest.java:36]:
```suggestion
  }

```

**Review by bion — APPROVED** [review]
Please add a code comment explaining why that code needs to be there

**Comment by shanemc-goog** [comment]
> Please add a code comment explaining why that code needs to be there

I don't believe this strays from any standard test setup such that it needs calling out.  EG @Before configures state before tests and typically sets unset class members, as is happening here.

Are you feeling like the class member needs documentation more so?

File: server/test/modules/MainModuleTest.java [hunk 3]
```diff
@@ -29,6 +30,11 @@ public class MainModuleTest {
   // This means that there's no need or a corresponding @Before method.
```
Comment (clouser2) [inline server/test/modules/MainModuleTest.java:30]: This can be removed

**Review by clouser2 — APPROVED** [review]

**Comment by bion** [comment]
> Are you feeling like the class member needs documentation more so?

ahhh maybe not, after looking at it more closely. feel free to ignore

File: server/test/modules/MainModuleTest.java (hunk 3 above)
Comment (shanemc-goog) [inline server/test/modules/MainModuleTest.java:30]:
```suggestion
```

Comment (shanemc-goog) [reply server/test/modules/MainModuleTest.java]: done

**Pull request merged**